bitbots_blackboard/src/bitbots_blackboard/capsules/game_status_capsule.py

```diff
@@ -16,7 +16,7 @@ def __init__(self):
         self.gamestate = GameState()
         self.last_update = 0
         self.unpenalized_time = 0
-        self.last_goal_from_us_time = rospy.Duration(86400)
+        self.last_goal_from_us_time = -10
```

Comment:
I think `-10` isn't the optimal value here. Because my understanding is that it implies a goal 10 seconds before the game. If we query the goals in the last 30 seconds at the start this fails. Something like `86400` would be better.